### ISSUE TITLE
Rename `Organization` to something more appropriate (semantically close to `Changemaker`).

### ISSUE THREAD

**Issue by slifty** [PhilanthropyDataCommons/service#1117]
We currently store an entity called `Organization` (as opposed to `Proposal`)

On call today @kfogel suggested we should make this more specific: `ChangeMaker`

This seems like a reasonable idea, especially as we are about to add new entity types that represent "organizations" (in a real world sense) and they are distinct from Organization/ChangeMaker.

Implementing this change would mean adjusting our endpoints in a breaking way; that's OK to do, but it is something we should try to only do once if we can help it.  The bottom line question to me is: "is ChangeMaker a forever term?"

---

EDIT: since opening we have honed in on `Applicant` instead of `Organization`

**Comment by kfogel** [comment]
Authoritative answer: "changemaker", with capitalization like a regular English word.  Internally we may use the "CM" abbreviation sometimes, but the public-facing word is "changemaker".

_(Source: Julie Hoffman [in Slack](https://coolteam5000.slack.com/archives/C0313DWR7FT/p1721762701053369) just now, replying to Karl: "changemaker with capitalization like any other English word. thanks!!")_

**Comment by slifty** [comment]
I am going to take this on now since it impacts the work for #1116

**Comment by slifty** [comment]
Short term update on this -- I'm pausing for a moment just to make 100% final sure we want `Changemaker` (vs, say, `applicant`).

The considerations here is that we don't want a term that might not last, since this has impact on API endpoints and data beyond just being used internally in a lot of ways.  I expect to hear back tomorrow!

In the meantime, back to provenance 😅

**Comment by jmergy** [comment]
I would lean more to keep it `Organization ` for now. Not something I would think we should try and mess with, for now, since changemaker is not super-universally known/understood.

**Comment by kfogel** [comment]
@jmergy What about `Applicant` or `Grantseeker`?  (This would only show up in internal code and in the API; it has no effect on what's presented on the front end.)

IMHO, `Organization` has become too generic.  We now have other organizational entities represented in the PDC, such as funders and data providers.  So we probably don't want to use "organization" internally for changemaker -- those other things are organizations too, so it would be confusing: why do we use the generic word "organization" for one kind of organization, while using more-specific words for other kinds of organizations?

We could solve this with `Applicant` or `Grantseeker` (technically, the letter is the most accurate, since not every grant happens as the result of an application process, but I think either one is fine).

Thoughts?

**Comment by kfogel** [comment]
Response (by email) from @jmergy :

> I understand the situation more now. Originally, I got the sense we were looking to change the "organization" references to "changemaker" since changemaker, and I agree it's a interesting term in PDC-work, is often used in context (i.e. an organization that is doing work in the sector and is funded through grants) but I didn't want to limit the term organization to only that in PDC world. If we are talking about this need, totally makes sense to me to use /applicant or something like that. I agree with you.

TL;DR: Let's use `applicant` internally!

**Comment by slifty** [comment]
One thought before I pull the code trigger: in our system an `Applicant` may exist without any actual applications (e.g. they would be in the system if they had IRS data that we loaded into PDC despite the lack of any actual proposals / having not actually applied to anything).

`Grantseeker` has a similar problem (an organization may not actually be seeking grants despite being in PDC), although it is slightly less explicit / one could argue that someone seeks grants despite not having submitted an application.

I still think `ChangeMaker` is a less intuitive name for an internal entity (even though it doesn't have this issue).

One term that came out of a brainstorm with a robot was `Grant Eligible Organization` -- this is nice because it's descriptive AND more specific than `Organization`, though it's obviously wordy.

---

I'll leave this here for a moment in case anybody has thoughts

**Comment by bickelj** [comment]
I'm curious if this change applies to all our Organization base fields. I assume so, right?

**Comment by bickelj** [comment]
@slifty Regarding your previous comment, an individual applying for a grant is not really an "organization" so I still like (the original) Applicant. It's shorter than the "Grant Eligible [X]" which I definitely would not like peppered throughout.

**Comment by slifty** [comment]
@bickelj yeah I hear that (GEO being long and wordy and organization specific).

That said... another example of the awkwardness of applicant / grantseeker though is that we have talked about the idea of being able to list partner organizations on a proposal -- e.g. an organization that isn't the `applicant` at all (and calling it an applicant could result in meaningful confusion because it's actually just a partner).

I hate to say this but I'm starting to think maybe `changemaker` isn't so bad after all 🫣

**Comment by slifty** [comment]
> I'm curious if this change applies to all our Organization base fields. I assume so, right?

Good question.  I could see us saying that organization base fields keep that label because they are describing organizational attributes (e.g. "number of employees" is an organizational concept; it just so happens the organization it is describing is the applicant, though it could also be describing an organizational partner, for instance.)

My tendency right now is that `organization` in the context of base field scope is reasonable / doesn't need to be changed as part of this rename.  This leaves us more room to not box us into changes that might cause trouble later on with other use cases (e.g. partners vs applicants vs co-applicants vs individual applicants, etc).

**Comment by slifty** [comment]
Hahah I appreciate Karl's rename here 😂 

Also yeah now that this has sat, I'm just going to move forward with the original `Changemaker` request!  I know this looks like we spent like 3 months just to go in a circle but the good news is most of that was subliminal processing cycles / not thinking about it at all...

The reason for Changemaker vs applicant / etc is articulated in detail above but I'll give the summary here:

1. `Applicant` / `Grantseeker` implies a role in the system which may not be accurate to a given entity (the group in question may not actually be an applicant, for instance).
2. `Organization` is not specific enough and also doesn't allow for the potential for individuals in future iterations of PDC.
3. `Nonprofit` is overly specific, as not all of these entities are necessarily nonprofits.

The english language does not appear to have a good word for "an entity that may, at some point, be associated with a request for funding in some way" and so creating one (e.g. `Changemaker`) is our best bet!  Luckily we didn't have to create it ourselves / PDC central already figured out this was a problem and addressed it.
